src/main/java/org/jenkinsci/test/acceptance/update_center/UpdateCenterMetadata.java

```diff
@@ -78,7 +79,11 @@ private void init() {
         List<PluginMetadata> set = new ArrayList<>();
         for (PluginSpec n : plugins) {
             PluginMetadata p = this.plugins.get(n.getName());
-            if (p==null) throw new IllegalArgumentException("No such plugin " + n.getName());
+            if (p==null) {
```

Comment:
If the collection contained `null` value the `#init()` would blow up so I am quite confident about it.